leaflet/leaflet.d.ts

```diff
@@ -1185,6 +1185,13 @@ declare namespace L {
         distance: number;
     }
 
+    export namespace DomEvent {
+        export function on(el: HTMLElement, types: string | Object, fn: Function, context?: any): typeof DomEvent;
```

Comment:
This should be split into two functions:

```
export function on(el: HTMLElement, types: string, context?: Object): typeof DomEvent;
export function on(el: HTMLElement, eventMap: {[eventName: string]: Function}, context?: Object): typeof DomEvent;
```

http://leafletjs.com/reference-1.0.0.html#domevent